client/src/components/ViewPostDialog.tsx

```diff
@@ -206,6 +208,39 @@ function CapacityBar(props: { maxCapacity: number }) {
   );
 }
 
+function LocationHandler(props: {
```

Comment:
```suggestion
function LocationMap(props: {
```

Reply:
LocationMap is already taken, it's the component imported from `LocationMap.tsx`